parser/proto.y

```diff
@@ -336,9 +336,6 @@ messageLiteralWithBraces : '{' messageTextFormat '}' {
 		fields, delims := $2.toNodes()
 		$$ = ast.NewMessageLiteralNode($1, fields, delims, $3)
 	}
-	| '{' '}' {
-		$$ = ast.NewMessageLiteralNode($1, nil, nil, $2)
-	}
```

Comment:
I think it was more clear this way. To eliminate ambiguity, instead of removing this, remove the empty production at line 359.

Reply:
Reverted this change and removed the empty production.